nav2_navfn_planner/include/nav2_navfn_planner/navfn_planner.hpp

```diff
@@ -25,7 +25,7 @@
 #include "geometry_msgs/msg/point.hpp"
 #include "geometry_msgs/msg/pose_stamped.hpp"
 #include "nav2_core/global_planner.hpp"
-#include "nav2_core/exceptions.hpp"
+#include "nav2_core/planner_exceptions.hpp"
```

Comment:
Why are there planner changes here?

Reply:
I renamed the exceptions to planner_exceptions to keep with the naming convention that I imposed. 

planner_exceptions
controller_exceptions